Cargo.toml

```diff
@@ -264,6 +264,7 @@ serde_html_form = { workspace = true, optional = true }
 serde_json = { workspace = true, optional = true }
 tokio = { workspace = true, features = ["macros", "io-std"], optional = true }
 tracing = { workspace = true, optional = true }
+aws-lc-sys = { version = "0.27", features = ["bindgen"] }
```

Comment:
I suppose you added it because we need the bindgen feature.

That said, this doesn't belong here.

1. add that dependency to the workspace dependencies instead
2. add it as a dependency (`workspace = true, features = ["bindgen"]`) to `rama-net` instead, and only when "rustls" is enabled, as it is not used for boring or if tls is not enabled at all

Reply:
Once done you'll also have to sort dependencies btw:

```
cargo sort --workspace --grouped
```

If you do not have that cmd yet, you'll need to install it: https://crates.io/crates/cargo-sort